config/locales/en.yml

```diff
@@ -130,7 +130,7 @@ en:
     title_example: Article title
     view: View Article
     view_all: View All
-    view_doc: The sum of all views to each article during the course term
+    view_doc: The sum of all views to each article during the program period
```

Comment:
Let's call this "project" instead of "program period". That will be general enough to work for both courses and programs.